plugin/kprom/config.go

```diff
@@ -101,6 +103,14 @@ func WithClientLabel() Opt {
 	return opt{func(c *cfg) { c.withClientLabel = true }}
 }
 
+// WithStaticLabel adds a static label to all metrics.
+func WithStaticLabel(label, value string) Opt {
```

Comment:
Is there a use case to a person wanting multiple static labels, such that this should be `labelvalues ...string)`?